lib/ioki/model/operator/constraints.rb

```diff
@@ -6,11 +6,31 @@ module Operator
       class Constraints < Base
         unvalidated true
 
-        attribute :max_passengers, on: :read, type: :integer
-        attribute :max_wheelchairs, on: :read, type: :integer
-        attribute :max_walkers, on: :read, type: :integer
-        attribute :max_storage_spaces, on: :read, type: :integer
-        attribute :area, on: :read, type: :object, class_name: 'Geojson'
+        # attribute :max_passengers,
+        #           on:   :read,
+        #           type: :integer
+
+        # attribute :max_wheelchairs,
+        #           on:   :read,
+        #           type: :integer
+
+        # attribute :max_walkers,
+        #           on:   :read,
+        #           type: :integer
+
+        # attribute :max_storage_spaces,
+        #           on:   :read,
+        #           type: :integer
```

Comment:
Why are these attributes commented out? If we don't need them, we can remove them and git can bring them back if we need to :)

Reply:
Yes, you're right. I kept them in case something changes. But after some testing I can confirm that these attributes are not needed at the moment.

Reply:
done!